sky/tools/create_macos_gen_snapshots.py

```diff
@@ -82,21 +82,9 @@ def generate_gen_snapshot(directory, destination):
     print('Cannot find gen_snapshot at %s' % gen_snapshot_dir)
     sys.exit(1)
 
-  command = [
+  subprocess.check_call([
```

Comment:
Reading the docs, it looks like `check_call` will cause this script to generate output even when it is successful (if xcrun does). If this is a script that we are running as part of the build, then when it is successful, it should generate no output.

Reply:
`xcrun bitcode_strip` generates no output on success.